app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/non_calculated_summary.scala.html

```diff
@@ -112,24 +112,27 @@
         )
     }
 
+   @if(!isATrust) {
```

Comment:
was this actually showing for trusts? It should have been that those answers would be `None` for trusts so that it wouldn't show